### DISCUSSION TITLE
Metadata names file location

### DISCUSSION THREAD

**Discussion by Majakhtar** [nftchef/art-engine#105] [Help + Q&A] [answered]
Hello,

Thank you for answering my questions, I really appreciate them!

Can I ask - In my metadata = Trait type will say the file location instead of the file name and the value will say the file name, right below this entry this will be duplicated with the correct trait and value.
 Thank you xx

Below you can see the 3rd trait shows file location and Gems is the actual files name and below that is the correct style, so this is duplicating this. 

[ "trait_type": "Background",
      "value": "Peach"
    },
    {
      "trait_type": "Base",
      "value": "XXXXX"
    },
    {
      "trait_type": "C:\\Users\\XXXXXX\\Desktop\\NFTs\\art-engine-nested-folder-structure\\layers",
      "value": "Gems"
    },
    {
      "trait_type": "Gems",
      "value": "Gold XXXXX",

**Comment by nftchef** [comment, marked as answer]
This issue is caused by having a non-weighted file in the root. More info on the solution in #73 
https://github.com/nftchef/art-engine/discussions/73#discussioncomment-2266165

**Reply by Majakhtar** [reply]
Amazing Thank you! x
